openlibrary/templates/account/books.html

```diff
@@ -100,6 +100,9 @@ <h3>
                 <li><a class="list-link" href="$(lst.key)">$lst['name'] ($(len(lst.seeds)))</a></li>
             </div>
           </ul>
+        <ul class="import-export-lists">
+          <li class="subsection"><a href="/account/import">$_("Import / Export")</a></li>
```

Comment:
```suggestion
          <li class="subsection">$_("Import / Export")</li>
          <li><a href="/account/import">$_("Import and export options")</a></li>
```
Prefer having the link not also be the subsection header.  The link should also have `text-decoration: underline` to indicate that patrons will be navigating away from the page when clicking the link.

Reply:
PR updated
![image](https://user-images.githubusercontent.com/64412143/123140110-8d7ce700-d474-11eb-9877-1ec0bf9b1677.png)